dropbox/1.0/modules/ROOT/pages/dropbox-connector-reference.adoc

```diff
@@ -0,0 +1,9718 @@
+| Response Timeout a| Number |  How long Mule waits for a response to complete before the response times out. |  |
+| Response Timeout Unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  Time unit for the *Response Timeout* field. |  |
+| Streaming Type a| Enumeration, one of:
+
+** AUTO
+** ALWAYS
+** NEVER |  How the connector streams request content to the remote service:
+
+* `AUTO` (default)
+
+Connector automatically uses the best streaming strategy based on the request content.
+
+* `ALWAYS`
+
+Connector always streams the request content.
+
+* `NEVER`
+
+Connector does not stream the request content. |  |
+| Target Variable a| String |  Name of the variable that stores the operation's output. |  |
+| Target Value a| String |  Expression that evaluates the operation's output. The expression outcome is stored in the target variable. |  #[payload] |
+| Reconnection Strategy a| * <<Reconnect>>
+* <<ReconnectForever>> |  Retry strategy in case of connectivity errors. |  |
+|===
+
+=== Output
+
+[%autowidth.spread]
+|===
+|Type |Any
+| Attributes Type a| <<HttpResponseAttributes>>
+|===
+
+=== For Configurations
+
+* <<Config>>
+
+=== Throws
+
+* DROPBOX-USERV2:BAD_REQUEST
+* DROPBOX-USERV2:CLIENT_ERROR
+* DROPBOX-USERV2:CONNECTIVITY
+* DROPBOX-USERV2:INTERNAL_SERVER_ERROR
+* DROPBOX-USERV2:NOT_ACCEPTABLE
+* DROPBOX-USERV2:NOT_FOUND
+* DROPBOX-USERV2:RETRY_EXHAUSTED
+* DROPBOX-USERV2:SERVER_ERROR
+* DROPBOX-USERV2:SERVICE_UNAVAILABLE
+* DROPBOX-USERV2:TIMEOUT
+* DROPBOX-USERV2:TOO_MANY_REQUESTS
+* DROPBOX-USERV2:UNAUTHORIZED
+* DROPBOX-USERV2:UNSUPPORTED_MEDIA_TYPE
+
+
+[[CreateFilesUploadSessionFinishBatch]]
+== Files Upload Session Finish Batch
+`<dropbox-userv2:create-files-upload-session-finish-batch>`
+
+
+This route helps you commit many files at once into a user's Dropbox. Use `upload_session/start` and `upload_session/append_v2` to upload file contents. Uploading many files in parallel can help increase throughput. Once the file contents have been uploaded, instead of calling `upload_session/finish`, use this route to finish all of your upload sessions in a single request. `UploadSessionStartArg.close` or `UploadSessionAppendArg.close` need to be true for the last `upload_session/start` or `upload_session/append_v2` call. The maximum size of a file one can upload to an upload session is 350 GB. This route will return a job_id immediately and do the asynchronous commit job in background. Use `upload_session/finish_batch/check` to check the job status. For the same account, this route should be executed serially, which means you should not start the next job before the current job finishes. You can allow up to 1000 entries in a single request. This operation makes an HTTP POST request to the `/files/upload_session/finish_batch` endpoint.
```

Comment:
```suggestion
This route enables you to commit many files simultaneously into a user's Dropbox. Use `upload_session/start` and `upload_session/append_v2` to upload file contents. Uploading many files in parallel can help increase throughput. After the file contents are uploaded, instead of calling `upload_session/finish`, use this route to finish all of your upload sessions in a single request. `UploadSessionStartArg.close` or `UploadSessionAppendArg.close` must be true for the last `upload_session/start` or `upload_session/append_v2` call. The maximum size of a file you can upload to an upload session is 350 GB. This route returns a job_id immediately and performs the asynchronous commit job in the background. Use `upload_session/finish_batch/check` to check the job status. For the same account, this route should be executed serially, which means you should not start the next job before the current job finishes. You can have up to 1000 entries in a single request. This operation makes an HTTP POST request to the `/files/upload_session/finish_batch` endpoint.
```